manifests/base/patch/patch-openshift.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: "Kepler exporter"
+    openshift.io/display-name: ""
+  name: kepler
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kepler-exporter
+  namespace: kepler
+spec:
+  template:
+    spec:
+      containers:
+      - name: kepler-exporter
+        volumeMounts:
+        - name: kernel-src
+          mountPath: /usr/src/kernels
+        - name: kernel-debug
+          mountPath: /sys/kernel/debug
+        securityContext:
+          privileged: true
+      volumes:      
+      - name: kernel-debug
+        hostPath:
+          path: /sys/kernel/debug
+          type: Directory
```

Comment:
There is still an open discussion if mounting /sys/kernel/debug is actually needed for OpenShift. In my local tests everything seemed to work perfectly fine and normally when excluding it. See the discussion [here](https://github.com/sustainable-computing-io/kepler-operator/pull/185/files#r1310982559) and the PR https://github.com/sustainable-computing-io/kepler-operator/pull/198. Maybe you can test this as well?

Reply:
I believe`/sys/kernel/debug` is still needed for libbpf to access the tracepoint files such as `/sys/kernel/debug/tracing/events/irq/softirq_entry`. Have you tried deploying with `latest-libbpf` tag?

Reply:
Yes I believe I tested both. Can probably try again later today or tomorrow! As I am no expert though maybe somebody else with more knowledge about libbpf could test as well? Don't know who that is though :)

Reply:
Actually, I am one of the team who works on this. 
However, I just recheck the manifest and found that there is no need to add the mount `/sys/kernel/debug` here because the `/sys` is already attached by kepler manifest and the path is the same (i.e., `/sys/kernel/debug`). Will remove it.

Reply:
@BGrasnick Thanks again. 
Made a rebase and update the change to open shift-patch.